concepts/felts/about.md

```diff
@@ -1 +1,26 @@
 # Felt Type
+
+# Scalar Types in Cairo
+
+Scalar types represent single values in Cairo. Cairo features three primary scalar types: **felts**, **integers**, and **booleans**. These types are similar to those in other programming languages but have unique characteristics and behaviors in Cairo.
+
+### Felt Type
+
+In Cairo, the default type for a variable or argument, if not explicitly specified, is a **field element**. This is represented by the keyword `felt252`. A field element is an integer within the range `0 ≤ x < `P, where `P` is a very large prime number currently equal to $2^{251} + 17 \cdot 2^{192} + 1$.
+
+#### Arithmetic Operations
+
+When performing arithmetic operations (addition, subtraction, multiplication) with field elements, if the result falls outside the range of the prime number `P`, an overflow or underflow occurs. To handle this, Cairo adjusts the result by adding or subtracting an appropriate multiple of `P` to bring the result back within the valid range. Essentially, this means the result is computed modulo `P`.
+
+#### Division in Cairo
+
+A critical distinction between integers and field elements is how division is handled. Unlike regular CPU integer division, where the result of `x` divided by `y` is defined as the integer part of the quotient (returning the floor value), Cairo ensures that the division of field elements always satisfies the equation:
+
+$(x / y) * y = x$
```

Comment:
Can you make all these divisions appear as fractions, similar to how you've done on [line 23](https://github.com/exercism/cairo/pull/56/files#diff-598032b99d9fffc7e22fbe18fe245d4c88c88a0044b7ba60b89b3688666dcab8R23) (the `\frac{P+1}{2}` part).